Userland/Libraries/LibGfx/ImageFormats/WebPLoaderLossy.cpp

```diff
@@ -1245,13 +1286,11 @@ ErrorOr<NonnullRefPtr<Bitmap>> decode_webp_chunk_VP8_contents(VP8Header const& v
 
     // Done with the first partition!
 
-    if (header.number_of_dct_partitions > 1)
-        return Error::from_string_literal("WebPImageDecoderPlugin: decoding lossy webps with more than one dct partition not yet implemented");
-
     auto bitmap_format = include_alpha_channel ? BitmapFormat::BGRA8888 : BitmapFormat::BGRx8888;
     auto bitmap = TRY(Bitmap::create(bitmap_format, { macroblock_width * 16, macroblock_height * 16 }));
 
-    TRY(decode_VP8_image_data(*bitmap, header, vp8_header.second_partition, macroblock_width, macroblock_height, macroblock_metadata));
+    auto data_partitions = TRY(split_data_partitions(vp8_header.second_partition, header.number_of_dct_partitions));
```

Comment:
Bit of a nitpick, but perhaps `second_partition` should be called `secondary_partitions` or similar to indicate that it contains multiple partitions?

Reply:
The spec alls it "second partition" :/

Reply:
Ah, looking back at this, it seems like I misunderstood that `second_partition` was referring to all the DCT token partitions, including the first.